backend-api/src/functions/asyncBiometricToken/asyncBiometricTokenHandler.ts

```diff
@@ -7,17 +7,30 @@ import {
   dependencies,
   IAsyncBiometricTokenDependencies,
 } from "./handlerDependencies";
-import { notImplementedResponse } from "../common/lambdaResponses";
+import {
+  badRequestResponse,
+  notImplementedResponse,
+} from "../common/lambdaResponses";
+import { validateRequestBody } from "./validateRequestBody/validateRequestBody";
 
 export async function lambdaHandlerConstructor(
   dependencies: IAsyncBiometricTokenDependencies,
-  _event: APIGatewayProxyEvent,
+  event: APIGatewayProxyEvent,
   context: Context,
 ): Promise<APIGatewayProxyResult> {
   const logger = dependencies.logger();
   logger.addContext(context);
   logger.log("STARTED");
 
+  const validateRequestBodyResult = validateRequestBody(event.body);
+  if (validateRequestBodyResult.isError) {
+    logger.log("REQUEST_BODY_INVALID", {
+      errorMessage: validateRequestBodyResult.value.errorMessage,
+    });
+    return badRequestResponse("invalid_request", "Request body invalid");
+  }
+  // const { sessionId, documentType } = parsedRequestBodyOrError.value
```

Comment:
Nitpick: comment can either be deleted or updated to reflect new variable names

Reply:
Gone